app/services/IdentityService.scala

```diff
@@ -110,4 +114,9 @@ class IdentityService(apiUrl: String, apiClientToken: String)(implicit wsClient:
         Left(s"Identity API error: ${requestHolder.method} ${uriWithoutQuery(requestHolder.uri)} STATUS ${r.status}")
     }
   }
+}
+
+trait IdentityServiceOrStub {
```

Comment:
I feel like calling this trait `IdentityServiceOrStub` unnecessarily leaks to the rest of the application that the id service might sometimes be a stub.  You could call the trait `IdentityService` and then the implementation `HttpIdentityService`, `RestIdentityService`, `IdentityServiceImpl`, or something else that distinguishes it from the stub?

Reply:
Yes, good point. I shall take that suggestion.